stubs/redis/redis/asyncio/client.pyi

```diff
@@ -13,10 +13,10 @@ from redis.typing import ChannelT, EncodableT, KeyT
 PubSubHandler: TypeAlias = Callable[[dict[str, str]], Awaitable[None]]
 
 class ResponseCallbackProtocol(Protocol):
-    def __call__(self, response: Any, **kwargs): ...
+    def __call__(self, response: Any, **kwargs) -> None: ...
```

Comment:
This should likely return `object` instead in order to be more permissive. Presumably the runtime just ignores whatever is returned.